gripper/core/CMakeLists.txt

```diff
@@ -1,4 +1,9 @@
 function(target_gripper_core TARGET)
+    if(${USE_SENSOR_MOVE})
+    target_compile_definitions(${TARGET} PUBLIC USE_SENSOR_MOVE)
+    target_compile_definitions(${TARGET} PUBLIC P_BUFF_SIZE=1700)
```

Comment:
lets not do this and remove all of the instances of USE_SENSOR_MOVE from the gripper code, if we never use it then we don't need to add the compile time changes for it

Reply:
Seth said it was ok to keep gripper capacitive data logging ability in firmware for future use, hidden behind the USE_SENSOR_MOVE flag